ci-operator/step-registry/assisted/baremetal/setup/assisted-baremetal-setup-ref.yaml

```
@@ -39,6 +39,6 @@ ref:
   - name: ENVIRONMENT
     default: "local"
     documentation: Choose environment to run the test over. Current options are "local" (the default) and "production".
-  timeout: 180m0s
+  timeout: 240m0s
```

let's have it simpler to read:
```suggestion
  timeout: 4h0m0s
```